src/Features/Core/Portable/ChangeSignature/ChangeSignatureCodeAction.cs

```diff
@@ -30,7 +30,11 @@ internal class ChangeSignatureCodeAction(AbstractChangeSignatureService changeSi
         public override object? GetOptions(CancellationToken cancellationToken)
             => AbstractChangeSignatureService.GetChangeSignatureOptions(_context);
 
-        protected override async Task<IEnumerable<CodeActionOperation>> ComputeOperationsAsync(object options, CancellationToken cancellationToken)
+        protected override Task<IEnumerable<CodeActionOperation>> ComputeOperationsAsync(object options, CancellationToken cancellationToken)
+            => throw ExceptionUtilities.Unreachable();
```

Comment:
note: in teh future, we likely want to make ComputeOperationsAsync in teh base class virtual (no longer 'abstract') and doc that you need to either override it, or the method below.

Reply:
we could also obsolete-it, pushing existing code to override the progress-takign api instead.